_health-care/_js/components/personal-information/Gender.jsx

```diff
@@ -1,45 +1,42 @@
 import React from 'react';
 import _ from 'lodash';
 
+/**
+ * Select component for getting a person's gender.
+ *
+ * No validation is provided since base UI elements provide all
+ * common options as provided in reference PDF/JS form.
+ */
 class Gender extends React.Component {
   constructor() {
     super();
-    this.state = { hasError: false };
     this.handleChange = this.handleChange.bind(this);
   }
 
   componentWillMount() {
-    this.id = _.uniqueId();
+    this.selectId = _.uniqueId('gender-select-');
   }
 
+  // handleChange(domEvent) {
   handleChange() {
-    const gender = this.refs.gender.value;
-
-    if (this.validate(gender)) {
-      this.setState({ hasError: false });
-    } else {
-      this.setState({ hasError: true });
-    }
-
-    this.props.onUserInput(gender);
+    this.props.onUserInput(this.refs.gender.value);
   }
 
   render() {
-    const errorClass = this.state.hasError ? 'usa-input-error' : '';
     return (
       <div>
-        <div className={`usa-input-grid usa-input-grid-large ${errorClass}`}>
-          <label htmlFor={`${this.id}-gender`}>
+        <div>
```

Comment:
I think you may still want to keep the grid classes, just not the errorClass. So: `className="usa-input-grid usa-input-grid-large"`